README.md

```diff
@@ -1,14 +1,20 @@
+# Sticky Notes
```

Comment:
The app name is just "Sticky". You can drop "Notes".

Reply:
resolved